### ISSUE TITLE
Extending Flight - unmapped/-registered methods

### ISSUE THREAD

**Issue by dvdvnl** [flightphp/core#203]
Extending using the _map_/_register_ method works like a charm. But if I call a method/object that has not been mapped/registered before, no exception is thrown.

E.g.

```
Flight::map('bar', function(){
    echo "bar!";
});

Flight::route('/', function(){
    Flight::bar();
    Flight::foo();
});

Flight::start();
```

produces the output

```
bar!
```

instead of throwing an error, stating that the method/object _foo_ has not yet been mapped/registered or something similar.

**Comment by ghost** [comment]
@dvdvnl This issue has been addressed in the following PR: https://github.com/mikecao/flight/pull/295 - I know it's been a while, I assume you no longer have the issue, but it needed fixing anyway.

**Comment by mikecao** [comment]
Fixed in b30d5e5fa2de08c3e91c1165ee226fd83f29ba9b
